New File.txt

```diff
@@ -0,0 +1 @@
+<HTML></HTML>
```

Comment:
Technically, opening and closing this tag right here ends the HTML document. So technically everything after this could be ignored. Most browsers will still try to render it though, but this is a syntax error.